frontend/app/pages/index.vue

```diff
@@ -10,45 +10,42 @@ const items = [
 </script>
 
 <template>
-  <UContainer>
-    <UCard class="mt-10">
-      <template #header>
-        <div class="flex justify-between">
-          <h1 class="text-2xl font-bold">
-            Welcome to Nuxt UI Starter
-          </h1>
-          <ColorScheme>
-            <USelect
-              v-model="$colorMode.preference"
-              :items="['system', 'light', 'dark']"
-            />
-          </ColorScheme>
+  <div class="px-4 py-4 dark:bg-black">
+    <div class="flex justify-center items-center h-screen">
+      <logo class="fill-current text-black dark:text-white max-h-96 max-w-96 mr-12" src="img/logo.svg" alt="Hack4Krak logo" />
+      <div class="text-black dark:text-white font-[25] text-[300px] flex justify-center items-center">
+        |
+      </div>
```

Comment:
```suggestion
      <USeparator orientation="vertical" class="h-88" size="lg" :ui="{ border: 'border-black dark:border-white'}"/>
```

I love this! It's so creative, but please, use better way :3